### PR TITLE
test ManagedCompletableFuture thenRun methods

### PR DESCRIPTION
**Pull request by njr-11**
Create test bucket for ManagedCompletableFuture.  Initially, include one test case with coverage of the runAsync/thenRun/thenRunAsync methods, part of which will need to be commented out until we properly tie in thread context propagation to our experimental ManagedCompletableFuture implementation.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.ws.concurrent_fat_rx/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/output
```
Comment (aguibert) [inline dev/com.ibm.ws.concurrent_fat_rx/.gitignore]: file can be removed

File: dev/com.ibm.ws.concurrent.rx/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Export-Package: \
+  com.ibm.websphere.concurrent.rx
+
+instrument.classesExcludes: com/ibm/ws/concurrent/rx/resources/*.class
+  
+-buildpath: \
+	com.ibm.websphere.appserver.spi.logging;version=latest,\
+	com.ibm.websphere.javaee.concurrent.1.0;version=latest,\
+	com.ibm.websphere.org.osgi.core;version=latest,\
+	com.ibm.ws.concurrent;version=latest,\
```
Comment (aguibert) [inline dev/com.ibm.ws.concurrent.rx/bnd.bnd]: do we need the dependency on `com.ibm.ws.concurrent;version=latest`?  Seems that just the API would be sufficient.

File: dev/com.ibm.ws.concurrent_fat_rx/fat/src/com/ibm/ws/concurrent/rx/FATSuite.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import componenttest.custom.junit.runner.AlwaysPassesTest;
+
+@RunWith(Suite.class)
+@SuiteClasses({
+                AlwaysPassesTest.class,
```
Comment (aguibert) [inline dev/com.ibm.ws.concurrent_fat_rx/fat/src/com/ibm/ws/concurrent/rx/FATSuite.java]: AlwaysPassesTest can be removed

File: dev/com.ibm.ws.concurrent_fat_rx/publish/files/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+/bundles
```
Comment (aguibert) [inline dev/com.ibm.ws.concurrent_fat_rx/publish/files/.gitignore]: this file can be removed

File: dev/com.ibm.ws.concurrent_fat_rx/publish/servers/com.ibm.ws.concurrent.fat.rx/.gitignore [hunk 5]
```diff
@@ -0,0 +1 @@
+apps/
```
Comment (aguibert) [inline dev/com.ibm.ws.concurrent_fat_rx/publish/servers/com.ibm.ws.concurrent.fat.rx/.gitignore]: this file can be removed

File: dev/com.ibm.ws.concurrent.rx/bnd.bnd [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+Bundle-Description: Completion Stage for Managed Executors, version ${bVersion}
+
+WS-TraceGroup: concurrent
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+Export-Package: \
+  com.ibm.websphere.concurrent.rx
+
+instrument.classesExcludes: com/ibm/ws/concurrent/rx/resources/*.class
```
[the remark this answers is not in the document]
Comment (njr-11) [reply dev/com.ibm.ws.concurrent.rx/bnd.bnd]: This project was copied from another, and it seems standard practice to disable instrumentation of NLS messages classes.  If that is no longer necessary to explicitly call out, let me know and I'll remove it.  I don't want to disable instrumentation of the whole project, which is why I didn't disable more generally.

File: dev/com.ibm.ws.concurrent.rx/bnd.bnd (hunk 2 above)
Comment (njr-11) [reply dev/com.ibm.ws.concurrent.rx/bnd.bnd]: We don't need it yet, but will, once we have updated the implementation to properly use context service/policy executor of the managed executor service instance.  This was preparing for the very near term future, so I will leave it here.

File: dev/com.ibm.websphere.appserver.concurrentRX-1.0/.gitignore [hunk 7]
```diff
@@ -0,0 +1,3 @@
+/bin/
```
[the remark this answers is not in the document]
Comment (njr-11) [reply dev/com.ibm.websphere.appserver.concurrentRX-1.0/.gitignore]: Removing this and all of the other files you noted that were unneeded - an unfortunate consequence of creating these projects by copying from existing projects.  Thanks for catching all of these.

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
The build for the first 2 commits (lacking the code review fix to remove unused files) was successful with all tests passing.

**Comment by njr-11** [comment]
The build with the code review updates had 2 test failures for pre-existing intermittent issues.

**Pull request merged**